localization/ro_RO.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+$labels = array();
+$labels['internal'] = 'Mesajul a fost intern';
+$labels['unencrypted'] = 'Mesaj primit pe o conexiune necriptată!';
+?>
```

Comment:
@liviuconcioiu does this translation look good ?

Reply:
I would do something like:

```PHP
$labels['unencrypted'] = 'Mesaj primit printr-o conexiune necriptată!';
```